scripts/skills/perks/perk_rf_second_wind.nut

```diff
@@ -0,0 +1,31 @@
+this.perk_rf_second_wind <- ::inherit("scripts/skills/skill", {
+	m = {
+		RecoverActionPointTarget = 4
+	},
+	function create()
+	{
+		this.m.ID = "perk.rf_second_wind";
+		this.m.Name = ::Const.Strings.PerkName.RF_SecondWind;
+		this.m.Description = ::Const.Strings.PerkDescription.RF_SecondWind;
+		// this.m.Icon = "ui/perks/rf_second_wind.png";		// TODO: add icon for this perk
+		this.m.Type = ::Const.SkillType.Perk;
+		this.m.Order = ::Const.SkillOrder.Perk - 5;
+		this.m.IsActive = false;
+		this.m.IsStacking = false;
+		this.m.IsHidden = false;
+	}
+
+	function onWaitTurn()
+	{
+		local actor = this.getContainer().getActor();
+
+		local recoveredActionPoints = this.m.RecoverActionPointTarget - actor.getActionPoints();
+		if (recoveredActionPoints > 0)
+		{
+			actor.setActionPoints(actor.getActionPoints() + recoveredActionPoints);
+			::Tactical.EventLog.log(::Const.UI.getColorizedEntityName(actor) + " recovers " + ::MSU.Text.colorGreen(recoveredActionPoints) + " Action Point(s)");
```

Comment:
This is minor but it would be nice to do `Action " + (recoveredActionPoints == 1 ? "Point" : "Points")` instead of `(s)`.

Reply:
I prefer the readability of this code (40 less character in that line) over the ever so slight improvement of the resulting text, and I prefer if this decision remains coders preference.

Reply:
There are ways to write it in a readable way in my opinion but honestly it's a minor thing. I see that vanilla also does this `(s)` thing in tooltips so I guess there is precedent for that already. So you can disregard my comment.